### PR TITLE
updates to 4.11 for 4.11.3.0 

### PR DESCRIPTION
**Pull request by PaulAngus**

### CODE REVIEW

**Comment by PaulAngus** [comment]
requesting docbuild

**Comment by rhtyd** [comment]
Thanks @PaulAngus I'll review the built contents tomorrow.

File: source/releasenotes/compat.rst [hunk 1]
```diff
@@ -94,20 +94,19 @@ best results, one of the following browsers recommended:
...
-1st June 2018. And will be removed from the CloudStack codebase from the first 
-release after 1st November 2018.
+The following hypervisors will no longer be supported in the next release
 
 -  XenServer 6.2
 -  XenServer 6.5
 -  vSphere 5.0
 -  vSphere 5.1
+-  vSphere 5.5
+-  CentOS (KVM) 6.x
```
Comment (rhtyd) [inline source/releasenotes/compat.rst]: +1!

**Comment by rhtyd** [comment]
@borisstoyanov @andrijapanic can you review as well?
Let's merge after there is one more review.

@PaulAngus can you review discussion here:
https://github.com/apache/cloudstack/issues/3450#issuecomment-507697560

**Pull request merged**